src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -1172,6 +1173,20 @@ internal StackFrameVersionOne CreateStackFrameVersionOne(StackFrame v2StackFrame
             return stackFrame;
         }
 
+        private static int HexToInt(string hexValue)
+        {
+            if (string.IsNullOrWhiteSpace(hexValue))
+            {
+                return 0;
+            }
+            // strip the leading 0x if found.
+            if (hexValue.StartsWith("0x", StringComparison.OrdinalIgnoreCase))
+            {
+                hexValue = hexValue.Substring(2);
+            }
+            return int.TryParse(hexValue, NumberStyles.HexNumber, CultureInfo.InvariantCulture, out int address) ? address : 0;
```

Comment:
>0 [](start = 125, length = 1)

this had to be zero and not -1, since thats what v1.0 supports and considers default value.